doc/user/content/ops/diagnosing-using-sql.md

```diff
@@ -136,6 +136,11 @@ group by mdo.id, mdo.name
 order by sum(mas.records) desc;
 ```
 
+We've also bundled a [memory usage visualization tool](https://materialize.com/docs/ops/monitoring/#memory-usage-visualization)
+to aid in debugging. The amount of memory used my Materialize should
```

Comment:
`The amount of memory used by Materialize`

Reply:
thanks!